imagePicker.py

```diff
@@ -29,12 +29,12 @@ def main():
         if Card_Query.lower() == 'clear' or Card_Query.lower() == '':
             # clear the image and loop
             print('Resetting image...')
-            shutil.copy2('blank.png', 'card.png')

```

Comment:
So the reason that that there are these placeholder blank cards is that the way OBS works, you need to point it at something for it to watch for updates.  If the file ever disappears or is deleted, OBS seems to complain and not check for updates and display the picture.  That's why the blank.png is a 1x1 transparent image, and instead of deleting the card.png, it is replaced with the 1x1 transparent one.

This also would allow for someone (if they wanted to) to show some other image when not searching up a card by just replacing the blank.png

So yes, I agree that it's cleaner the way you suggested, but the way that OBS works seems to necessitate that there always be an image there, and copying from a reference 1x1 transparent png is a lot easier than image manipulation by code.

Reply:
Aaaah... so that's why you didn't nuke the file. I clearly didn't read the full README (specially the ADDITIONAL INFO). 
Sorry for that rogue change in the functionality, as my changes didn't intend to affect it.